test/verify/check-storage-lvm2

```diff
@@ -33,22 +34,21 @@ class TestStorage(StorageCase):
         self.login_and_go("/storage")
         b.wait_in_text("#drives", "VirtIO")
 
-        m.add_disk("50M", serial="DISK1")
-        m.add_disk("50M", serial="DISK2")
-        b.wait_in_text("#drives", "DISK1")
-        b.wait_in_text("#drives", "DISK2")
-
-        dev_1 = "/dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_DISK1"
-        dev_2 = "/dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_DISK2"
+        dev_1 = self.add_ram_disk()
+        dev_2 = self.add_loopback_disk()
+        b.wait_in_text("#drives", "scsi_debug")
+        # loop devices don't appear in the drives list
 
         # Create a volume group out of two disks
         m.execute("vgcreate TEST1 %s %s" % (dev_1, dev_2))
+        # just in case the test fails
+        self.addCleanup(m.execute, "vgremove --force TEST 2>/dev/null || true")
         b.wait_in_text("#devices", "TEST1")
         b.wait_in_text("#devices", "/dev/TEST1/")
         b.click('tr:contains("TEST1")')
         b.wait_visible("#storage-detail")
-        b.wait_in_text("#detail-sidebar", "DISK1")
-        b.wait_in_text("#detail-sidebar", "DISK2")
+        b.wait_in_text("#detail-sidebar", "scsi_debug")
```

Comment:
Let's wait for `dev_1` here, and in other places.

Reply:
I can change the tests to wait for /dev/sda in `#drives`. But here in particular that doesn't work, as `#detail-sidebar` does *not* show the /dev/* names, just the vendor/model name.

Reply:
Ah, it only shows the device name for loop backs?  Good point.

Reply:
Done in most places, and I also added waiting for the loop device, and added an extra commit that changes the waiting for all other tests.